Objects/genobject.c

```diff
@@ -273,7 +270,6 @@ gen_send_ex2(PyGenObject *gen, PyObject *arg, PyObject **presult,
     _PyErr_ClearExcState(&gen->gi_exc_state);
 
     gen->gi_frame_state = FRAME_CLEARED;
-    _PyFrame_Clear(frame);
```

Comment:
Actually it makes little difference for users of PEP 523.
There are two types of users of PEP 523.
* Those that tweak the frame in some way and call `_PyEval_EvalDefault()`. It makes no difference to them as they did no cleanup anyway.
* Those that reimplement the interpreter (I suspect this type is mythical). They need to fully re-implement everything for every release anyway.

`PyEval_EvalFrame` and `PyEval_EvalFrameEx` are not changed because they take frame objects. So when the frame is to be cleared, the frame object takes ownership of it.